design/authenticators-status/authn-status-new-implem.md

```diff
@@ -1,65 +1,84 @@
-# Implementing Authenticator Status Checks
-The authenticator status check allows operators to check if the authenticator is configured correctly. With this capability, 
-operators can make a RESTful request using the Conjur API and receive a response identifying if configuration was successful.
-If not, the operators receive the proper error identifying the step of failure. 
-The first authenticator to utilize this feature is Authn OpenID Connect. 
+# Implementing Authenticator Status API
+The authenticator status API allows operators to get immediate feedback on the authenticator configuration. Once the configuration of an authenticator is complete, the operator sends a status request and the status of the given authenticator is returned. 
 
-_For a complete status check flow [diagram](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn-status-flow.jpeg)_
+If the configuration was successful, the response received will reflect this. If not, an error will be provided in the response, identifying the step of failure.
 
-This doc describes how to expand the status check functionality to other authenticators.
+The first authenticator to utilize this feature xis Authn OpenID Connect. For more information about the design behind the Status API and implementation details see the [Status epic](https://github.com/cyberark/conjur/issues/1062). 
+It is advised that you read the epic before expanding status to other authenticators.
+
+_For a complete status flow [diagram](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn-status-flow.jpeg)_
+
+This doc describes how to expand the status functionality to other authenticators.
 For each authenticator, a suite of [general](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn_status_general.md) 
 checks are run followed by those specific to the authenticator.  
 
 General checks include:
-1. The requesting user has access to the authenticator status route webservice
-1. _The authenticator implements the status check_
+
+**NOTE:** Steps that require developer intervention to implement and enable status for authenticators are italicized below
```

Comment:
please don't add notes under a sentence that finishes with ":"

Reply:
it is very inconvenient to read

Reply:
forgot that one. good catch. Reworking the sentence